### ISSUE TITLE
[Gallery][iOS]Email Manager - When clicking on 'compose' button, the email send screen is quickly disappearing

### ISSUE THREAD

**Issue by sakshi173** [unoplatform/uno#11181]
### Current behavior

![eMailManager-iOS Issue](https://user-images.githubusercontent.com/74563602/216475244-1873a3ee-2f0e-44a0-96d9-d607f1350b83.PNG)

https://user-images.githubusercontent.com/74563602/216475257-3069d76d-8284-462d-9b5f-a81439e36e4d.MP4

### Expected behavior

The email compose screen should not disappear until click on cancel or send button.

### Works on UWP/WinUI

None

### Affected platforms

iOS

**Comment by jeromelaban** [comment]
@sakshi173 are you testing in a simulator or on a device? If on a device, are emails configured on that device?

**Comment by sakshi173** [comment]
@jeromelaban Yes tested this on device. yes email is configured on the device,  I am getting the composed email screen but its quickly disappering (not gettting chance to review or make changes and clicking on send button.)
![image](https://user-images.githubusercontent.com/74563602/216993351-527f7c9d-6406-4a0e-89db-c609738f50d4.png)

**Comment by MartinZikmund** [comment]
We can verify this is happening only in Gallery, or in general. In case this is Gallery only, the issue can be transferred there

**Comment by jeromelaban** [comment]
@sakshi173 I wonder, do you have the "Mail" app setup on your device? It sometimes happens when the phone is not setup completely.

**Comment by agneszitte** [comment]
> @sakshi173 I wonder, do you have the "Mail" app setup on your device? It sometimes happens when the phone is not setup completely.

@sakshi173 I added this issue to the "Needs QA Review" list so you can review it please

@jeromelaban added to the current release backlog for @sakshi173 to review it but can move to other backlog depending on the outcome and priority

**Comment by sakshi173** [comment]
@agneszitte and @jeromelaban  Still I am seeing the same issue, Not sure what need to change in the device, otherwise for all other app the mail things working fine, please anyone else can also try that and confirm about this defect. Below is the latest video of the issue.

https://github.com/unoplatform/uno/assets/74563602/c4636591-0d13-4fae-8cb1-10bf6b189c28

**Comment by agneszitte** [comment]
Thanks a lot for the feedback @sakshi173 we will ask the Team to investigate.

> We can verify this is happening only in Gallery, or in general. In case this is Gallery only, the issue can be transferred there

For anyone taking this issue, please first check if it is a Gallery-only issue.
- If so we will move the issue to the Uno.Gallery repo
- If not, we will keep it to the Uno repo

**Comment by MartinZikmund** [comment]
Likely still relevant for Skia

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** This is an iOS-specific issue involving the system email compose screen (`MFMailComposeViewController`) disappearing immediately. It requires an iOS device with a configured email account and cannot be reproduced with a runtime test or manual sample in the test infrastructure.
